### ISSUE TITLE
[Settings]: Screen Reader is not announcing Breadcrumb control's name throughout Settings page.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#13826]
### Windows Terminal version

1.15.2003.0

### Windows build number

10.0.25179.1000

### Other Software

**Test Environment:**
OS: Window 11 22H2 OS Build 25179.1000
App: Windows Terminal Preview
AT: Narrator and Voice Access

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Voice Access AT.
3. Inside Settings> Defaults >Appearance page, navigate to Defaults control and observe the issue.
4. Navigate to other breadcrumb control throughout the Terminal app and observe the issue.

**User Experience:**
People with Visual impairments like Screen Reader will be impacted here as well as they will not be able to know the purpose of control.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[Unnammed Breadcrumb control.zip](https://github.com/microsoft/terminal/files/9401733/Unnammed.Breadcrumb.control.zip)

### Expected Behavior

Screen Reader should announce Breadcrumb control's name throughout Settings page.

For 'Defaults' Control in 'Defaults> Appearance', Screen Reader is announcing 'Defaults, link, Breadcrumb bar item 1 of 2'

### Actual Behavior

Screen Reader is not announcing Breadcrumb control's name throughout Settings page.

For 'Defaults' Control in 'Defaults> Appearance', Screen Reader is announcing 'Breadcrumb bar item 1 of 2'

**Observation:** Issue repro with JAWS and NVDA Screen Reader.

**Comment by v-rpundir** [comment]
Hi @carlos-zamora In Windows Terminal v1.15.2874(mentioned build in above comment) it is fixed but In Window Terminal Preview Version: 1.16.2642.0(mentioned Store Download link) it is not fixed, Is that fine?

Also let us know for all other bugs whether we have verify on Normal, Preview or both.

**Comment by DHowett** [comment]
@v-rpundir We will release the Preview update soon. We have been focusing on making Stable version better for the Windows 11 22H2 update. Thanks for testing. :)

**Comment by Shubham786786** [comment]
Closing the bug as issue is fixed on the latest terminal build.
